src/views/Widget/TabContent/Panes/Operate/OperatePane.tsx

```diff
@@ -99,6 +99,16 @@ const OperatePane: React.FC<OperatePaneProps> = (props) => {
     handleInputChange(formValues);
   }, [formValues, handleInputChange]);
 
+  useEffect(() => {
+    if(isLoading){
+      reloadCrucibles();
```

Comment:
@Manevle16 Could we do `reloadCrucibles().then(() => setLoading(false))` and remove the second useEffect